os/include/tinyara/fs/ioctl.h

```diff
@@ -433,6 +434,14 @@
 #define MMINFOIOC_PARSE             _MMINFOIOC(0x0002)
 #define MMINFOIOC_MNG_ALLOCFAIL     _MMINFOIOC(0x0003)
 
+/* Compress driver ioctl definitions ************************/
+#define _COMPIOCVALID(c)    (_IOC_TYPE(c) == _COMPBASE)
+#define _COMPIOC(nr)        _IOC(_COMPBASE, nr)
+
+#define COMPIOC_COMPRESS           _COMPIOC(0x0001)
+#define COMPIOC_TYPE		   _COMPIOC(0x0002)
```

Comment:
1. It's difficult to know what it operates.
Let's rename it like `COMPIOC_GET_COMP_TYPE` or something else.
2. Please add code comment for return value. If we get return value with 1, how can we know what it means?
3. Same as other definitions. Need any description.

Reply:
done